packages/accounts-password/password_tests.js

```diff
@@ -1789,6 +1789,26 @@ if (Meteor.isServer) (() => {
       ]);
     });
 
+    Tinytest.addAsync("passwords, replace email", async test => {
+      const origEmail = `originalemail@test.com`;
+      const userId = await Accounts.createUser({
```

Comment:
I understand that on the server, [`createUser` is essentially the same as `createUserAsync`](https://github.com/meteor/meteor/blob/ae8cf586acc9a4c7bf9a5ab79dc5f8b7ef433a64/packages/accounts-password/password_server.js#L1291), but this naming convention diverges from the usual sync/async pattern. Would it be better to use `createUserAsync` here in case we update the naming later?